src/openapi/streaming/streaming.js

```diff
@@ -148,6 +148,9 @@ function onConnectionStateChanged(change) {
             break;
 
         case this.CONNECTION_STATE_RECONNECTING:
+            if (signalRTransport && signalRTransport.name === 'longPolling') {

```

Comment:
There is one issue with this approach. It will log also all reconnect occurrences withing longPolling session. 
Are we interested in those as well or only scenarios where we transition from webSocket to longPolling transport?

Reply:
Now only logs when transport changes (so no initial or reconnecting attempts).